guides/source/active_job_basics.md

```diff
@@ -124,8 +124,68 @@ GuestsCleanupJob.perform_later(guest1, guest2, filter: 'some_filter')
 
 That's it!
 
+### Bulk Enqueuing
+
+You can Enqueue multiple jobs at once using [`perform_all_later`][]. Bulk Enqueuing reduces the number of round trips to a queue data store like Redis.
+
+`perform_all_later` is a top-level api on ActiveJob. It accepts instantiated jobs as arguments (note that this is different from `perform_later`). `perform_all_later` does call `perform` under the hood. The arguments passed to `new` will be passed on to `perform` when it's eventually called.
```

Comment:
(suggestion, non-blocking): I found the order of these to be slightly confusing and took a stab at re-ordering it a bit. 

```suggestion
`perform_all_later` is a top-level api on ActiveJob that allows you to enqueue multiple jobs at once. Bulk enqueuing reduces the number of round trips to a queue data store like Redis.

`perform_all_later` accepts instantiated jobs as arguments (note that this is different from `perform_later`), and it calls  `perform` under the hood by passing the arguments from `new` to `perform`.
```

Reply:
In addition, would be useful to say what are the benefits of enqueuing multiple jobs or why you'd want to do that?